ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbarManager.java

```diff
@@ -84,6 +85,13 @@ public void setTitle(ReactToolbar view, @Nullable String title) {
     view.setTitle(title);
   }
 
+  @ReactProp(name = "rtl")
+  public void setLayoutDirection(ReactToolbar view, Boolean flag) {
+    if (flag.equals(true)) {
```

Comment:
Exactly, it shouldn't just do anything. Imagine you have a component that renders:
   
    if (this.state.rtl !== undefined) return <ToolbarAndroid rtl={this.state.rtl}/>;
    else return <ToolbarAndroid/>;

Imagine your app's logic can update `this.state.rtl` freely between 3 values: `undefined`, `false` and `true`.

With the current implementation once you set the value of `rtl` to `true` it could never been updated. This is because your update method does nothing when value other than `true` is provided. It *should* handle any sort of updates as I mentioned before.